docs/content/latest/integrations/prisma.md

```diff
@@ -306,27 +308,27 @@ node index.js
 
 ## Clean up
 
-Now that you're done with this exploration, you can clean up the pieces for your next adventure.
+Now that you have completed with this exploration, you can clean up the pieces as follows:
 
-1. Stop the YugabyteDB cluster
+Stop the YugabyteDB cluster:
 
 ```sh
 ./bin/yb-ctl stop
 ```
 
-To completely remove all YugabyteDB data/cluster-state you can instead run:
+To completely remove all YugabyteDB data/cluster-state, you can instead run the following command:
 
 ```sh
 ./bin/yb-ctl destroy
 2. Stop The Prisma container
 docker stop <container-id>
 ```
 
-To completely remove all Prisma data/tate you can additionally run:
+To completely remove all Prisma data/state, you can additionally run the following command:
 
 ```sh
 docker rm <container-id>
 ```
 
-Note: you can list running containers by running `docker ps`.
+To list running containers, execute `docker ps`.
```

Comment:
```suggestion
To list running containers, run `docker ps`.
```

Reply:
Don't like two runs in s row. "execute" is better here.